Include containernetworking-plugins in ubuntu deps

smartin015 opened the pull request:
Not including the containernetworking-plugins package on Ubuntu Server 20.04 LTS gives the following error when building and running from source:

```
WARN[0001] Failed to load cached network config: network podman not found in CNI cache, falling back to loading network podman from disk 
WARN[0001] 1 error occurred:
        * plugin type="tuning" failed (delete): failed to find plugin "tuning" in path [/usr/local/libexec/cni /usr/libexec/cni /usr/local/lib/cni /usr/lib/cni /opt/cni/bin]
 
Error: plugin type="bridge" failed (add): failed to find plugin "bridge" in path [/usr/local/libexec/cni /usr/libexec/cni /usr/local/lib/cni /usr/lib/cni /opt/cni/bin]
```

After `sudo apt -y install containernetworking-plugins`, this message goes away and I can sucessfully start a container.

lsm5 commented on getting-started/installation.md:
```diff
@@ -44,15 +56,36 @@ sudo pacman -S podman
+Podman is available in the default Extras repos for CentOS 7 and in
+the AppStream repo for CentOS 8 and Stream, however the available version often
+lags the upstream release.
+
```
need to add a `yum -y install podman` here..

TomSweeneyRedHat commented:
LGTM, thanks @lsm5!

smartin015 committed:
Include containernetworking-plugins in ubuntu deps

Not including the containernetworking-plugins package on Ubuntu Server 20.04 LTS gives the following error when building and running from source:

```
WARN[0001] Failed to load cached network config: network podman not found in CNI cache, falling back to loading network podman from disk 
WARN[0001] 1 error occurred:
        * plugin type="tuning" failed (delete): failed to find plugin "tuning" in path [/usr/local/libexec/cni /usr/libexec/cni /usr/local/lib/cni /usr/lib/cni /opt/cni/bin]
 
Error: plugin type="bridge" failed (add): failed to find plugin "bridge" in path [/usr/local/libexec/cni /usr/libexec/cni /usr/local/lib/cni /usr/lib/cni /opt/cni/bin]
```

Luap99 commented:
For newer podman versions we should use `netavark` and `aardvark-dns` and only fall back to cni if that is not installed.
So I think it would make more sense to list those over the cni plugins.

TomSweeneyRedHat commented:
@smartin015 thanks for the PR.  @lsm5 just updated this same document pretty significantly.  I'm not sure you got his changes or not.  Lokesh, thoughts?  

I concur with @Luap99 , we're moving away from CNI, we should talk about netavark/aardvark-dns unless we're talking about old versions of the code (Podman 3.* and below)

lsm5 commented:
hmm, do we even have nv / av on 20.04 LTS? https://launchpad.net/ubuntu/+source/netavark

Maybe we'll need separate sections for nv/av and cni ?

lsm5 commented:
@smartin015 could you please revise your PR to add a separate section for ubuntu 20.04 below the current ubuntu section? We would prefer not to mention cni unless required. Thanks.

lsm5 commented:
@smartin015 also, please sign your commit to get the DCO task successful. See: https://github.com/containers/podman.io/pull/201/checks?check_run_id=16176897846

lsm5 commented:
closing this in favour of #206.
